src/entity/notifications/NotificationAttachmentV2.ts

```diff
@@ -20,4 +20,7 @@ export class NotificationAttachmentV2 {
 
   @Column({ type: 'text' })
   referenceId: string;
+
+  @Column({ type: 'bool', default: false })
+  isAttachmentVideo: boolean;
```

Comment:
May I suggest utilizing the attachment type instead? It already exists
`isAttachmentVideo` is very specific

Reply:
We currently use the type whether it is a `post` or something, but we can probably do that. Seeing we haven't really utilized the attachment type in the FE.

Reply:
@rebelchris @omBratteng thoughts?

Reply:
The problem with the current approach is that it's very specific. Tomorrow we would want to add a podcast for example, you will have to introduce another boolean field.

Reply:
Agreed, let me refactor the part we generate post attachments and identify whether it was a video or not.